aclk/aclk_query.c

```diff
@@ -638,6 +663,39 @@ void aclk_query_threads_start(struct aclk_query_threads *query_threads)
     }
 }
 
+/**
+ * Checks and updates popcorning state of rrdhost
+ * returns actual/updated popcorning state
+ */
+
+ACLK_POPCORNING_STATE aclk_host_popcorning(RRDHOST *host)
```

Comment:
This function has the same name as the `#define` above, kind of confusing.

Reply:
pushed a few commits that should make the names a bit more human friendly. hopefully :)